1-getting-started-lessons/1-intro-to-programming-languages/README.md

```diff
@@ -122,9 +122,9 @@ Developers rely on editors for a few additional reasons:
 
 ### Browsers
 
-Another crucial tool is the browser. Web developers rely on the browser to observe how their code runs on the web, it's also used to view visual elements of a web page that are written in the editor, like HTML.
+Another crucial tool is the browser. Web developers rely on the browser to see how their code runs on the web, it's also used to display the visual elements of a web page that are written in the editor, like HTML.
```

Comment:
This is actually a run-on sentence, so let's add a period after web and start a new sentence with It's